fs/io_zenfs.h

```diff
@@ -140,6 +142,11 @@ class ZoneFile {
   IOStatus Recover();
 
   void ReplaceExtentList(std::vector<ZoneExtent*> new_list);
+  IOStatus SetLinkInfo(std::string linkfile);
+  IOStatus RemoveLinkInfo(std::string linkfile);
+  uint32_t GetNrLinks() { return nr_links_; }
+  void SetNrLinks(uint32_t nrLinks) { nr_links_ = nrLinks; }
```

Comment:
I think the `SetNrLinks` method should not exist because `nr_links_` is implicitly set by `SetLinkInfo` and `RemoveLinkInfo`. (Also see comments on `SetNrLinks` references)

Reply:
You are right, SetNrLinks() is redundant at this point, will remove it,